index.ts

```diff
@@ -1,16 +1,17 @@
 'use strict';
 
+
+
 const makeErrorChecker = (e: object) => {
     type T = typeof e;
 
+    let errorHandler = Symbol('errorHandler')
```

Comment:
Hehe, nice use of a Symbol. The idea is to "hide" the properties from the end user, yeah? Good idea.

Reply:
Yes man. That was the idea (privacy)